### PR TITLE
Added definition for an upcoming Pimoroni EInk board

### PR DESCRIPTION
**Pull request by ZodiusInfuser**
I think I've set everything up okay, but let me know if there's any issues. It's been a while since I've added a new board def

### CODE REVIEW

File: ports/raspberrypi/boards/pimoroni_badger2040/board.c [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+#include "supervisor/board.h"
+
+void board_init(void) {
+}
```
Comment (tannewt) [inline ports/raspberrypi/boards/pimoroni_badger2040/board.c:30]: Want to init the eink here? See the magtag for an example that does.

**Review by tannewt — COMMENTED** [review]
Looks fine. One question.

File: ports/raspberrypi/boards/pimoroni_badger2040/board.c (hunk 1 above)
Comment (ZodiusInfuser) [reply ports/raspberrypi/boards/pimoroni_badger2040/board.c:30]: Thanks. I'll check with Gadgetoid, as he's been writing our support for this board, so will have a better idea of how feasible that is.

Comment (ZodiusInfuser) [reply ports/raspberrypi/boards/pimoroni_badger2040/board.c:30]: Checked with him and he agrees its a good idea to do, and in theory _shouldn't_ be too much work. However, we have a hard deadline for launch and looking at our outstanding task list, I don't know if it is feasible for us to get it implemented in time.

Would it be problematic to launch with these current definitions and add the board_init stuff at a later date?

Comment (tannewt) [reply ports/raspberrypi/boards/pimoroni_badger2040/board.c:30]: Not at all. Later is fine with me. Just wanted to make sure it was on your radar.

**Review by tannewt — APPROVED** [review]
Thank you!

**Pull request merged**